src/Header/Header.css

```diff
@@ -0,0 +1,10 @@
+header {
+  text-align: center;
+  background: green;
+}
+
+.heading {
+  margin: 0;
+  padding: 1em;
+  color: rgb(74, 35, 110);
+}
```

Comment:
Is it important to have an empty line at the end of js files?

Reply:
I don't think it is.